google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -434,7 +434,7 @@ interfaces:
     max_retry_delay_millis: 60000 # 60 seconds
     initial_rpc_timeout_millis: 12000 # 12 seconds
     rpc_timeout_multiplier: 1
-    max_rpc_timeout_millis: 12000 # 12 seconds
+    max_rpc_timeout_millis: 30000 # 30 seconds
```

Comment:
This config is only used for the `publish` method on the Publisher. Subscriber StreamingPull uses https://github.com/googleapis/googleapis/pull/411/files#diff-1445f68867b94d222bea9502a59f7c39R69. If I recall correctly from an internal discussion a while ago, we also wanted to change that; let me amend this PR.